src/TBuffer.cpp

```diff
@@ -2550,209 +2694,82 @@ inline int TBuffer::wrap(int startLine)
                     lineText = "";
                     hasContent = false;
                     // Start the next line by indent
-                    for (int i3 = 0; i3 < mWrapIndent; ++i3) {
-                        TChar pSpace;
+                    for (int i3 = 0; i3 < indentSize; ++i3) {
+                        TChar pSpace = format;
```

Comment:
:+1: Well spotted - I suspect this is why previously, wrapped text (when there was a non-zero indent) was being preceded by black spaces instead of those in the correct background colour... :grinning:

Reply:
actually everywhere that pass in "format" is actually just passing in a generic pSpace, so this wouldn't fix that.  Now I am interested in fixing that problem, but I believe it isn't related to this PR, so as such I would do a separate PR for that.

Reply:
Hmm I may not be grasping the problem discussed here, but is it really an issue?

Reply:
I think what the code "should do", is figure out what the format of the indent "should be".  Now that's what I would need to find out, which is what is a generic space supposed to look like?

Reply:
> actually everywhere that pass in "format" is actually just passing in a generic pSpace, so this wouldn't fix that. Now I am interested in fixing that problem,...

I am not sure that is the case - this method `TBuffer::wrapLine(...)`  is used in more than just `TConsole::luaWrapLine(int)` and in those other cases (`TConsole::printCommand(...)`, `TConsole::insertText(...)`, `TConsole::insertLink(...)`) a format is supplied...